Vagrantfile

```diff
@@ -2,27 +2,33 @@
 Vagrant::configure("2") do |config|
 
   # configure the basebox
-  config.vm.box = "tknerr/ubuntu1604-desktop"
-  config.vm.box_version = "2.0.27.1"
+  config.vm.box = "fasmat/ubuntu1804-desktop"
+  config.vm.box_version = "1.1.0"
+  config.vm.box_check_update = false
+  config.vm.guest = :ubuntu
 
   # set the hostname
   config.vm.hostname = "linus-kitchen.local"
+
   # don't create a new keypair
   config.ssh.insert_key = false
 
   # virtualbox specific customizations
   config.vm.provider :virtualbox do |vbox, override|
-    vbox.customize ["modifyvm", :id,
-      "--name", "Linus Kitchen",
-      "--memory", 4096,
-      "--cpus", Etc.nprocessors
-    ]
+    vbox.customize ["modifyvm", :id, "--name", "Linus Kitchen"]
+    vbox.customize ["modifyvm", :id, "--memory", 8192]
```

Comment:
I was tempted to increase to 8GB earlier, but then switched back to 4GB as I was mostly using the docker provider inside. But to honestly run a Windows Virtualbox VM inside, you definitely need the 8GB ;) 👍